ticket_tags_values.view.lkml

```diff
@@ -1,12 +1,23 @@
 view: ticket__tags {
   sql_table_name: zendesk.tickets__tags ;;
 
+  dimension: pkid {
+    primary_key: yes
+    type: string
+    hidden: yes
+    sql: (${TABLE}._sdc_level_0_id::text) || '_' || (${TABLE}._sdc_source_key_id::text) ;;
+  }
+
   dimension: ticket_id {
     type: number
+    hidden: yes
     sql: ${TABLE}._sdc_source_key_id ;;
   }
 
   dimension: value {
+    label: "Ticket Tag"
+    group_label: "Custom Fields"
+    description: "Custom tag associated with a ticket"
```

Comment:
I'm not sure why we are making this change. It looks like this currently:

![image](https://user-images.githubusercontent.com/171817/36400349-3e13048a-1585-11e8-81a2-42a58335fb5e.png)

If we want to rename, we can call it "Tag", or is this being optimized for a different view? Also why are we rolling it up under "Custom Fields"?

Reply:
What matters was the group label - I was thinking of putting it along other custom fields because these tags looked like custom input to me.  If we are happy with the current set up in the above screenshot, then I can close this PR.